client.js

```diff
@@ -26,5 +26,30 @@ on('onClientResourceStart', (resource) => {
                 }
             }, 1500);
         }
+        console.log('[snow] snow enabled');
+    }
+});
+
+on('onResourceStop', (resource) => {
+    if (resource === 'snow') {
+        ClearOverrideWeather();
+        SetSnowLevel(-1.0);
+
+        // Calling this native seems to crash the script no matter how it's used. It's only used 4 times in the whole game, it could be just buggered.
+        //UnloadCloudHat('Snowy 01', 1); 
+
+        SetForcePedFootstepsTracks(false);
+        SetForceVehicleTrails(false);
+        SetRainLevel(0.0);
```

Comment:
going off the [FiveM docs for this native](https://docs.fivem.net/natives/?_0x643E26EA6E024D92), it seems to me that this should be set to -1? 

> With an level of 0.0f rain and rain sounds are disabled and there won't be any new puddles.
> To use the rain level of the current weather, call this native with -1f as level.

I just feel like the desired behaviour of disabling a weather override would be to set the rain back to being controlled by the weather preset, which it already is when the resource starts so I guess even removing this call altogether would be less likely to cause issues (unless there's a specific reason for setting it to 0)